docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -48,12 +48,22 @@ yarn deploy --network network_name
 
 ## 4. Verify your smart contract
 
-You can verify your smart contract on Etherscan by running:
+In **Hardhat** you can verify your smart contract on Etherscan by running:
 
 ```
 yarn verify --network network_name
 ```
 
+In **Foundry** there isn't a specific command to verify, in this case you need to **Deploy and Verify** within the same command.
+
+```
+yarn deploy:verify --network sepolia
+```
+
+:::info Note
+Currently this command is kind of unstable for custom configured networks but works great for `sepolia` and `goerli`.
+:::info Note
```

Comment:
We can merge this now and remove `note`, Since foundry also now follows `yarn verify --network network_name`  after  https://github.com/scaffold-eth/scaffold-eth-2/pull/489  🙌